TramsDataApi/UseCases/UpdateAcademyConversionProject.cs

```diff
@@ -28,7 +29,9 @@ public AcademyConversionProjectResponse Execute(int id, UpdateAcademyConversionP
                 return null;
             }
 
-            var academyConversionProject = _tramsDbContext.AcademyConversionProjects.SingleOrDefault(p => p.IfdPipelineId == id) ??
+            var academyConversionProject = _tramsDbContext.AcademyConversionProjects
+                                               .Include(p => p.ProjectNotes)
```

Comment:
I dream of this with the current legacy data :p

Reply:
I think going back to the comments above this should be separate from the ACP use case and controller. It should be possible to add a project note to the database without having to pull out an ACP record as we already have the ACP Id in the http request